File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-0.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> minHeap = new PriorityQueue<>((a, b) -> Integer.compare(a, b));
        for (int n : nums){
            minHeap.offer(n);
            if (minHeap.size() > k){
                minHeap.poll();
            }
        }
        return minHeap.peek();
    }
}

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n = nums.size(), k = 1;
        int uIdx = 0, i = 1, j = n - 1;
        while (i <= j) {
            if (nums[i] != nums[uIdx]) {
                nums[uIdx + 1] = nums[i];
                uIdx++;
                k++;
            }
            i++;
        }
        return k;
    }
};